lib/ansible/plugins/connection/httpapi.py

```diff
@@ -202,7 +202,8 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
             self.httpapi = httpapi_loader.get(self._network_os, self)
```

Comment:
@ganeshrn ideally the `httpapi_loader` should also be called from fqcn

Reply:
The name of the sub-plugin need to be updated as fqcn which I have done in the latest commit.